test/github-issues/2984/issue-2984.ts

```diff
@@ -0,0 +1,25 @@
+import "reflect-metadata";
+
+import {closeTestingConnections, reloadTestingDatabases, setupSingleTestingConnection} from "../../utils/test-utils";
+import {createConnection, Connection} from "../../../src";
+
+describe("github issues > #2984 Discriminator conflict reported even for non-inherited tables", () => {
+    let connection: Connection;
+
+    before(async () => connection = await createConnection(setupSingleTestingConnection("postgres", {
```

Comment:
Why test it on just one database if fix is not database type specific?

Reply:
Whoops, that was a copy/paste from another test. I should be able to push a change for that tomorrow if you want.

Reply:
looks like in this case he doesn't need to do it since he don't really use connection. He just needs to check entity validation process.